Create vending machine

LOG91 opened the pull request:
first PR

Commit:
Create vending machine

first PR

crongro commented on index.js:
```diff
@@ -0,0 +1,81 @@
+        result.push(item);
+      }
+    }
+    return `사용가능한 음료수 목록 => ${result.join(', ')}`;
```
출력을 담당하는 함수를 만들어서 모든 출력을 거기서 하게 해보세요.

crongro commented on index.js:
```diff
@@ -0,0 +1,81 @@
+
+  // 음료를 고르는 함수
+  function selectedItem(item) {
+    if (beverageData[item].stock <= 0) return `${item}는 재고가 없습니다.`;
```
함수안에서 메시지를 분리해볼래요?

crongro reviewed (changes requested):
리뷰보고 수정해보세요.

Commit:
Add message object, add (var result)  before return

1. 메세지 객체를 만들어서 함수마다 입력되어 있던 메세지들을 모아놨습니다
2. 모든 함수에서 리턴하기 전에 result변수를 만들어 담은 다음에 return result;했습니다

Commit:
Create lotto.js

First commit

crongro commented on index.js:
```diff
@@ -34,6 +34,24 @@
   let accumCoin = 0;
 
+  // 메세지 객체
+  let message = {
```
시도는 좋은데요. 
message객체가 속성으로도 접근되고 어떤건 함수형태로 접근하는게 일관되지 않아서 쓸때 헷갈릴 듯.
모두다 함수로 하는 것은 어때요?

Commit:
Modify message object string to function

Commit:
Lotto first commit

커밋을 단계별로 하는 것을 깜빡했네요ㅠ.ㅠ

crongro reviewed (approved):
잘 하셨네요.
message객체는 message를 관리하기보다는 일종의 view역할을 하는거랑 비슷한거 같아요. 이런식으로 다음에도 객체화 해서 분리하면 좋겠네요.
